adminguide.rst

```diff
@@ -320,6 +322,52 @@ job queue and record of past jobs.
     If this happens it is best to stop Flux, remove the content.sqlite file,
     and restart.
 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+flux-accounting configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In order to support fairshare and multi-factor priority ordering of user jobs, the flux-accounting package must be present and configured for the system instance. This allows for the management of users and banks on a cluster, job priorities to be calculated using multiple factors and uses job record information from users to calculate and maintain fairshare values.
+
+The first thing to configure when first setting up the flux-accounting database is to set the PriorityUsageResetPeriod and PriorityDecayHalfLife parameters. Both of these parameters represent a number of weeks by which to hold usage factors up to the time period where jobs no longer play a factor in calculating a usage factor. If these parameters are not passed when creating the DB, PriorityDecayHalfLife is set to 1 week and PriorityUsageResetPeriod is set to 4 weeks, i.e the flux-accounting database will store up to a month's worth of jobs broken up into one week chunks:
```

Comment:
Is there no other database initialization required?  Add users? Associate banks to users?

Just a thought but maybe we could add subsections here for 1) configure the database for the first time,  2) create TOML config file, and 3) set up cron jobs

Reply:
> Is there no other database initialization required? Add users? Associate banks to users?

Other than specifying those two optional parameters, I don't believe there is any other database initialization required prior to adding banks and users. The command just below this text is enough to create a flux-accounting DB. 

Perhaps it would be helpful to add a small subsection that provides an example on adding a couple of banks and then adding a couple users to those banks? Along with adding those subsections like you mentioned?

Reply:
It's the adding banks and users that I thought maybe should be described here?  Or should that be in another document?

What happens if the admin follows all the instructions here, but doesn't add users and banks?   Can the system run jobs?